src/components/general/AddStakeholderModal/StakeholderList/styles.scss

```diff
@@ -0,0 +1,55 @@
+.widget {
+    display: flex;
+    position: relative;
+    flex-basis: calc(calc(100% / 3) - calc(var(--dui-spacing-small) * 2));
+    flex-direction: column;
+    flex-grow: 0;
```

Comment:
We should not add `flex-grow` / `flex-shrink` from here, that job belongs to its parent